triage/update_summaries.sh

```diff
@@ -39,7 +39,10 @@ bq --headless --format=json query -n 1000000 \
     job,
     number
   from
-    [k8s-gubernator:build.week]" > triage_builds.json
+    [k8s-gubernator:build.all]
```

Comment:
I suspect that this will increase the query time substantially. That may not be an issue though?
/lgtm
/hold

Reply:
That was my concern as well.  Locally it was happy, let's get some runs in and see what testgrid says